style.css

```diff
@@ -105,3 +105,413 @@ body {
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.third-sec {
+  width: 375px;
+  height: 1901px;
+  left: 0;
+  top: 4137px;
+  background: #1c1a19;
+}
+
+.third-sec-div {
+  padding: 0 24px;
+  height: 590px;
+  background-color: #1c1a19;
+  background-image: url("./images/illustration-about-me-mobile.png");
+  background-position: right 95%;
+  background-repeat: no-repeat;
+}
+
+.third-tittle {
+  width: 327px;
+  height: 79px;
+  padding-top: 84px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.third-p {
+  width: 327px;
+  height: 148px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-top: 96px;
+}
+
+.get-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+  margin-top: 12px;
+}
+
+.line2 {
+  border: 1px solid #313030;
+  margin-bottom: 70px;
+
+  /* padding: 1px; */
+  height: 1px;
+  width: 325px;
+}
+
+.line1 {
+  border: 1px solid #fff;
+  margin-top: 200px;
+}
+
+#padding {
+  margin-top: 100px;
+}
+
+.head {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  padding-top: 30px;
+}
+
+.head-items {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.lang {
+  font-family: 'InaiMathi', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.head-items a {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
+}
+
+.head-items li {
+  padding-top: 12px;
+}
+
+.li-p {
+  padding-top: 30px;
+}
+
+.css-btn:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.html-btn:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.boot-btn:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.ruby-btn:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.pro-html:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.pro-boot:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.pro-rubby:hover {
+  background-color: #ff6b00;
+  color: #fff;
+  transition-duration: 2s;
+}
+
+.pro-btn:hover {
+  color: #ff6b00;
+  background-color: #fff;
+  transition-duration: 2s;
+}
```

Comment:
- [OPTIONAL] Please create a border for the button when hovering following the Figma template. Check the other buttons for the same issue. As not requested by the previous reviewers, I consider that optional.